client/src/components/Form/FormElement.vue

```diff
@@ -255,6 +270,11 @@ library.add(faExclamation, faTimes, faArrowsAltH, faCaretSquareDown, faCaretSqua
                 :min="attrs.min"
                 :type="type"
                 :workflow-building-mode="workflowBuildingMode" />
+            <FormRadio
+                v-else-if="props.type == 'select' && attrs.display == 'radio'"
```

Comment:
As discussed out of band, it'd be cleaner for each input type to have a single toplevel component that is dispatched to here, where it can do its own logic on what type of component to render and use, whether it's a FormRadio or a Multiselect, etc.  The model just maps through.

Reply:
I added a `FormSelection` component mapping through. It did require a considerable amount of additional code but hopefully it helps when adding additional select variations.